test/AutoRest.TestServerLowLevel.Tests/body-array.cs

```diff
@@ -113,7 +113,7 @@ public class BodyArray : TestServerLowLevelTestBase
         [Test]
         public Task GetArrayBooleanWithNullExample() => Test(async (host) =>
         {
-            Response response = await new ArrayClient(Key, host, null).GetBooleanInvalidNullAsync();
+            Response response = await new ArrayClient(Key, host, null).GetBooleanInvalidNullAsync(new());
```

Comment:
It is better to specify the type, `new RequestContext()` in our test.

Reply:
The fact that `new()` might lead to ambiguity problem in the future doesn't mean we should change it to `new RequestContext()`. The reason is the same as why we accept `new()` to introduce ambiguity, i.e., we don't discourage user to  use `new()` just because that might lead to ambiguity when they introduce their own overloads.